austrakka/components/tree/__init__.py

```diff
@@ -18,31 +22,73 @@ def tree(ctx):
     ctx.context = ctx.parent.context
 
 
-@tree.command('add')
-@click.argument('newick', type=click.File('rb'))
-@opt_analysis()
-def tree_add(newick: BufferedReader, analysis: str):
-    '''Upload tree to AusTrakka'''
-    add_tree(newick, analysis)
+tree.add_command(version) if show_admin_cmds() else None
 
 
 @tree.command('list')
-@opt_analysis()
+@opt_project(required=True)
+@opt_show_disabled()
 @table_format_option()
-def tree_list(analysis: str, out_format: str):
+def tree_list(project: str, show_disabled: bool, out_format: str):
     '''List trees in AusTrakka'''
-    list_trees(out_format, analysis)
+    list_trees(project, show_disabled, out_format)
+
+
+@tree.command('add')
+@opt_abbrev()
+@opt_name(help='Tree Name')
+@opt_description()
+@opt_project()
+@opt_is_active()
+def tree_add(
+        abbrev: str,
+        name: str,
+        description: str,
+        project: str,
+        is_active: bool,
+):
+    """Add tree in AusTrakka"""
```

Comment:
Suggesting more explicit help text on these commands to make clear tree vs tree version:
"""Add a new named tree to a project. If a newick file is supplied, it will be uploaded to this tree entry, otherwise the tree entry will appear in the project with no tree versions until one is added. To add a newick file to an existing named tree, use the `tree version add` command."""